src/main/java/com/puppycrawl/tools/checkstyle/checks/UniquePropertiesCheck.java

```diff
@@ -84,6 +85,7 @@ protected void processFiltered(File file, List<String> lines) {
         for (Entry<String> duplication : properties
                 .getDuplicatedKeys().entrySet()) {
             final String keyName = duplication.getElement();
+            final List<String> lines = fileText.getLines();
```

Comment:
please avoid usage of temporal object - List<String> lines , that is easy refactoring.